Resources/config/templating.xml

```diff
@@ -44,7 +44,7 @@
 
         <service id="liip_theme.theme_auto_detect" class="%liip_theme.theme_auto_detect.class%" />
 
-        <service id="liip_theme.active_theme" class="%liip_theme.active_theme.class%">
+        <service id="liip_theme.active_theme" class="%liip_theme.active_theme.class%" public="true">
```

Comment:
I would not make this service public. Users can inject it properly if they want to make use of it. And since they will get a nice deprecation message with Symfony 3.4 they are also able to prepare their applications for this change.

Reply:
👍